### DISCUSSION TITLE
How to export suppression list & bounce list

### DISCUSSION THREAD

**Discussion by XenyFord** [postalserver/postal#3038] [Help with using Postal]
Dear Colleagues,

I'm using the latest version of Postal, and I have TWO different mail servers (em-server-1 & em-server-2) on it, and I would like to export the list of **suppression emails**, **bounced emails**, as well as **pending emails** Postal directly either from the command prompt or front end.

Can anyone please be kind enough to help me with how to get this done? Thank you in advance!

**Comment by willpower232** [comment]
You would probably need to query the databases directly to get that information, Postal doesn't really export information in bulk like this as it isn't designed to store information for very long.

**Reply by XenyFord** [reply]
Hi Mr. Will,

Could you please help me what are the exact commands to run in ubuntu to export the list of bounced or hardfail emails?

When I try to access mySQL DB using below command it gave me an error as below:

`root@postal:~# mysql -u root -p -h 127.0.0.1`

`Command 'mysql' not found, but can be installed with:`
`apt install mysql-client-core-8.0     # version 8.0.25-0ubuntu0.20.10.1, or`
`apt install mariadb-client-core-10.3  # version 1:10.3.29-0ubuntu0.20.10.1`

And I stuck here, and afraid to install mysql-client. So, my humble request you to pelase help me achieve this. Thank you in advance!

**Comment by XenyFord** [comment]
Hi Sir,
Could you please let me know the exact set of commands to run in Ubuntu SSH panel to achieve these, please? Thank you in advance!
